docs/docs/network-api-migration.md

```diff
@@ -0,0 +1,697 @@
+            "name": "network-name-1"
+        }
+    ]
+}
+</pre>
+</td>
+
+<td>
+  <pre>
+{
+    “container”: {
+      "portMappings": [
+        {
+          "containerPort": 0,
+          "name": "a-port-name",
+          “networkNames”: [“network-name-1”]
+        },
+        {
+          "containerPort": 0,
+          "name": "another-port-name",
+          “networkNames”: [“network-name-2”]
+        }
+      ]
+    },
+    "networks": [
+        {
+            "mode": "container",
+            "name": "network-name-1"
+        },
+        {
+            "mode": "container",
+            "name": "network-name-2"
+        }
+    ]
+}
+</pre>
+</td>
+</tr>
+</tbody>
+</table>
+
+## Bridge Network
+
+The following table summarizes the API transformations when using the network API in Bridge Mode:
+
+<table class="table">
+  <tbody>
+  <tr>
+  <th>
+    <b>1.4 API</b>
+  </th>
+  <th>
+    <b>1.5 API</b>
+  </th>
+  </tr>
+  <tr>
+    <td>
+      <pre>
+{
+    "container": {
+        "docker": {
+            "network": "BRIDGE",
+            "portMappings": [
+              {
+                "containerPort": 0,
+                “hostPort”: 0,
+                "name": "port-name"
+              }
+            ]
+        }
+    }
+    "ipAddress": {
+        "networkName": "network-name"
+    }
+}
+
+</pre>
+</td>
+
+<td>
+ <pre>
+
+ {
+    “container”: {
+      "portMappings": [
+        {
+          "containerPort": 0,
+          “hostPort”: 0,
+          "name": "port-name"
+        }
+      ]
+    },
+    "networks": [
+        {
+            "mode": "container/bridge"
+        }
+    ]
+}
+</pre>
+</td>
+</tr>
+</tbody>
+</table>
+
+**Changes from 1.4 to 1.5**
+
+- `ipAddresses` : Removed.
+- `container.docker.network` : Removed.
+- `container.docker.portMappings` : Moved to `container.portMappings`.
+- `networks` : Added.
+- `networks[x].mode` : Is “container/bridge” for bridge network.
+
+### Important Notes
+
+- An application cannot use more than one “container/bridge” network.
+- An application cannot mix networking modes: you must specify a single host network, a single container/bridge network, or one or more container networks.
+
+
+## Host Network
+
+The following table summarizes the API transformations when using the network API in Host Mode:
+
+<table class="table">
+  <tbody>
+  <tr>
+  <th>
+    <b>1.4 API</b>
+  </th>
+  <th>
+    <b>1.5 API</b>
+  </th>
+  </tr>
+  <tr>
+    <td>
+      <pre>
+{
+    "container": {
+        “type”: “DOCKER”,
+        "docker": {
+            “image”: “foo”,
+            "network": "HOST"
+        }
+    }
+    "portDefinitions": [
+        {
+            "name": "port-name",
+            "protocol" ...,
+            "port": ...
+        }
+    ]
+}
+</pre>
+</td>
+
+<td>
+ <pre>
+ {
+    “container”: {
+        “type”: “DOCKER”,
+        “docker”: {
+            “image”: “foo”
+        }
+    },
+    "networks": [
+        {
+            "mode": "host",
+            "name": "network-name"
```

Comment:
name is ignored for host networks, and it's confusing to see here. please remove